WebService/Startup.cs

```diff
@@ -55,6 +55,7 @@ public Startup(IHostingEnvironment env)
 
             if (ConfigFile.GetDevOnlyConfigFile() != null)
             {
+                Console.WriteLine("===========================\nLOADING SETTINGS FROM " + ConfigFile.GetDevOnlyConfigFile() + "\n===========================");
```

Comment:
do we need this console log?

Reply:
this is executed only when running the code on a local dev machine, ie not in prod. It is very useful to see this because the custom appsettings path is an env var which could affect solutions in different path. For instance, one might clone the repo many times into:

C:\dev\simulation
C:\workspace\simulation
C:\workspace\simulation_test

but the env var (used by GetDevOnlyConfigFile) is going to be the same, so a dev might be confused without knowing where configuration settings are loaded from.